packages/theme/src/cli/utilities/theme-environment/storefront-renderer.ts

```diff
@@ -102,8 +103,11 @@ function buildStorefrontUrl(session: DevServerSession, {path, sectionId, query}:
     params.append(key, value)
   }
 
+  // The Section Rendering API takes precendence over the Block Rendering API.
   if (sectionId) {
     params.append('section_id', sectionId)
+  } else if (appBlockId) {
```

Comment:
I expected updating an `app block` (targetting a section) to attach and `appBlockId` but for the `sectionId` to take precedence.
It seems like we only send `app_block_id` when it's an app embed being updated. If that's intended maybe we can change this to something like appEmbedBlockId?

Non-blocking.

Reply:
I had the _same_ conflict — but this was my reasoning: the API accepts the `app_block_id` as a parameter, so we should keep that consistent here, since the `storefront-renderer.ts` works as a client of the SFR API. This way, if in the future SFR supports rendering for app blocks as well, this logic wouldn’t be impacted. However, the logic in the hot reload module would be, as that’s where we’re distinguishing between section blocks and embed blocks.